templates/1_set_up_toffy.ipynb

```diff
@@ -15,9 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<p>Any reason why the <code>normalize</code> functions can't be combined into a single <code>normalize.calibrate_from_sweep(sweep_path, moly_masses, ...)</code>wrapper function?  This could also handle the <code>norm_func.json</code> saving as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yeah agreed</p>